omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_10/DrugSearchByMappingHandler1_10.java

```diff
@@ -36,6 +36,7 @@
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.stereotype.Component;
 
+
```

Comment:
Don't format classes you have not made changes too, can you get rid such changes please?